examples/otel-logs-routing/otel-collector-config.yml

```diff
@@ -0,0 +1,87 @@
+        disable_compression: false
+        # HTTP timeout when sending data. Defaults to 10s.
+        timeout: 10s
+        # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
+        # For this demo, we use a self-signed certificate on the Splunk docker instance, so this flag is set to true.
+        insecure_skip_verify: true
+    splunk_hec/logs2:
+        # Splunk HTTP Event Collector token.
+        token: "00000000-0000-0000-0000-0000000000000"
+        # URL to a Splunk instance to send data to.
+        endpoint: "https://splunk:8088/services/collector"
+        # Optional Splunk source: https://docs.splunk.com/Splexicon:Source
+        source: "output"
+        # Splunk index, optional name of the Splunk index targeted.
+        index: "logs2"
+        # Maximum HTTP connections to use simultaneously when sending data. Defaults to 100.
+        max_connections: 20
+        # Whether to disable gzip compression over HTTP. Defaults to false.
+        disable_compression: false
+        # HTTP timeout when sending data. Defaults to 10s.
+        timeout: 10s
+        # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
+        # For this demo, we use a self-signed certificate on the Splunk docker instance, so this flag is set to true.
+        insecure_skip_verify: true
+    splunk_hec/routing:
+        # Splunk HTTP Event Collector token.
+        token: "00000000-0000-0000-0000-0000000000000"
+        # URL to a Splunk instance to send data to.
+        endpoint: "http://localhost:8088/services/collector"
+        # Optional Splunk source: https://docs.splunk.com/Splexicon:Source
+        source: "output"
+        # Splunk index, optional name of the Splunk index targeted.
+        index: "logs2"
+        # Maximum HTTP connections to use simultaneously when sending data. Defaults to 100.
+        max_connections: 20
+        # Whether to disable gzip compression over HTTP. Defaults to false.
+        disable_compression: false
+        # HTTP timeout when sending data. Defaults to 10s.
+        timeout: 10s
+        # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
+        # For this demo, we use a self-signed certificate on the Splunk docker instance, so this flag is set to true.
+        insecure_skip_verify: true
+    logging:
+        loglevel: debug
+        sampling_initial: 5
+        sampling_thereafter: 200
+processors:
+    batch:
+
+extensions:
+    health_check:
+        endpoint: 0.0.0.0:13133
+    pprof:
+        endpoint: :1888
+    zpages:
+        endpoint: :55679
+
+service:
+    extensions: [ pprof, zpages, health_check ]
+    pipelines:
+        logs:
+            receivers: [ filelog ]
+            processors: [ batch ]
+            exporters: [ splunk_hec/logs, splunk_hec/logs2, splunk_hec/routing ]
```

Comment:
If logs2 were moved to the routing pipeline it may be more informative than using a logging exporter?

Reply:
A big ask has been to send data to 2 different Splunk instance, so logs and logs2 show this. The routing is extra to show we can route to another exporter. I debated creating another collector instead of sending to the same collector.

Reply:
OK I made the example even more complex with 2 otel collectors talking to each other.